runtime/browser-demo/browser-demo.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+//let runtime = require("../runtime.js");
+
+let Arc = require("../arc.js");
```

Comment:
At some point we should make in-browser Mocha tests to cover code that doesn't get exercised in node. For that reason, I thought it made sense to leave `browser-test` there and make a new folder for this entry point.